goenv.rb

```diff
@@ -0,0 +1,30 @@
+class Goenv < Formula
+  desc "Go version management"
+  homepage "https://github.com/syndbg/goenv"
+  url "https://github.com/syndbg/goenv/archive/20161028.tar.gz"
+  sha256 "3f283537c2b4f64a7549a009361ac4da6115be6b2c45462b008a58f8366a8804"
+  head "https://github.com/syndbg/goenv.git"
+
+  bottle :unneeded
+
+  def install
+    inreplace "libexec/goenv", "/usr/local", HOMEBREW_PREFIX
```

Comment:
I can try that, but this formula is simply following the pattern set by the `pyenv` and `rbenv` formula.